data/Liu2024Longitudinal.yaml

```diff
@@ -0,0 +1,1648 @@
+# yaml-language-server: $schema=.schema.yaml
+title: Longitudinal Fecal Shedding of SARS-CoV-2, Pepper Mild Mottle Virus, and Human
+  Mitochondrial DNA in COVID-19 Patients
+doi: 10.1101/2024.04.22.24305845
+description: >
+  The authors measured SARS-CoV-2, pepper mild mottle virus (PMMoV), and human mitochondrial
+  DNA (mtDNA) in longitudinal stool samples collected from 42 COVID-19 patients for
+  up to 42 days after the first sample collection date. Abundances were quantified
+  using Digital PCR assays targeting the N1 gences. The data was contributed by the
+  corresponding author.
+analytes:
+  stool_SARSCoV2_N1:
+    description: >
+      SARS-CoV-2 RNA genome copy concentration in stool samples. The concentration
+      were quantified in gene copies per dry weight of stool.
+    specimen: stool
+    biomarker: SARS-CoV-2
+    gene_target: N1
+    limit_of_quantification: 1000
+    limit_of_detection: unknown
+    unit: gc/dry gram
+    reference_event: confirmation date
+  stool_PMMoV:
+    description: >
+      PMMoV genome copy concentration in stool samples. The concentration were quantified
+      in gene copies per dry weight of stool.
+    specimen: stool
+    biomarker: PMMoV
```

Comment:
Are there specific targets for PMMoV and mtDNA?

Reply:
There are no specific targets for PMMoV and mtDNA in the preprint. I could ask my colleague about it though.

Reply:
Maybe we can merge this as-is and follow up at a later point if we need it?

Reply:
Sure, let's do it. I have emailed my colleague to ask for the information of gene targets for PMMoV and mtDNA.